src/g_game.c

```diff
@@ -368,6 +368,18 @@ static void G_DemoSkipTics(void)
   }
 }
 
+static double PitchToShearingHeight(double pitch)
+{
+  // Scale up to BAM and convert to radians. Clamp to sane range.
+  pitch *= FRACUNIT * M_PI / ANG180;
+  pitch = BETWEEN(-1.3, 1.3, pitch);
+
+  // Convert angle to y-shearing height and apply aspect ratio correction.
+  pitch = (160.0 * tan(pitch)) * (lookdirmax / 100.0) / 1.2;
```

Comment:
Oh, is it possible to avoid floating point trigonometric functions in code that's called multiple times per frame?

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1372/commits/735afb25ebb64f6f49a7e6e58da946a89db922b3